ajax/libs/colresizable/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "colresizable",
+  "filename": "colResizable-1.6.min.js",
```

Comment:
Based on the above reason, please rename `colResizable-1.6.min.js` to `colResizable.min.js` and update `filename` accordingly.